web/src/pages/billing/components/BatchGrid.tsx

```diff
@@ -344,26 +343,6 @@ const BatchCard: React.FC<{ item: AnalysisCostRecordBatch }> = ({ item }) => {
 const BatchGrid: React.FunctionComponent<{
     data: AnalysisCostRecord
 }> = ({ data }) => {
-    const [openRows, setOpenRows] = React.useState<string[]>([])
```

Comment:
Is this change expected?

Reply:
Good question! I think this code was left over from a previous version or something, none of it was actually used anywhere.